client/src/saga/county/uploadBallotManifestSaga.ts

```diff
@@ -43,9 +43,21 @@ function* uploadBallotManifestNetworkFail(): IterableIterator<any> {
     notice.danger('Network error: failed to upload ballot manifest.');
 }
 
+function* deleteFileOK(): IterableIterator<any>{
+    notice.ok('Successfully deleted file');
+}
+
+function* deleteFileNOTOK(): IterableIterator<any>{
+    notice.danger('There was an error deleting the file, please try again.');
+}
+
 function createUploadingBallotManifest(uploading: boolean) {
     function* uploadingBallotManifest(action: any): IterableIterator<any> {
         const data = { uploading };
+        // delete file uses this
+        if (action.data.fileType && action.data.fileType != 'bmi') {
+            return;
+        }
```

Comment:
This code makes it feel like the deletion actions might have been better off mirroring the upload actions (e.g. `DELETE_CVR_...` and `DELETE_BALLOT_MANIFEST...`) and being pulled out of the `uploadBallotManifestSaga`.

Reply:
I would say that the upload actions would have been better off mirroring the delete file action.